documents/operating.md

```diff
@@ -29,15 +29,67 @@ SECTION 2: 🤝🤝🤝 Partnership 🤝🤝🤝
+- [x] IP (Intellectual Property / Internet Protocol)
+- [x] Open source
 - [ ] When a project should be considered retired
 - [ ] Removing a project from BIFFUD
 - [ ] Adopting IP from existing entities / projects
 - [ ] Reallocating copyright / IP upon spinoff
 
+### Types of Project
+A. **Grant Funded:** projects funded by grants from third parties are executed
+by the BIFFUD member who has led the relationship with the funders, or by an
+alternative BIFFUD member as delegated by that applicant.  Money associated with
+a grant is to be distributed as per the grant's agreement.
+
+B. **Member Owned:** projects that are led or adopted by BIFFUD members.  The
+owners of a project must all be BIFFUD members, and the percentages and specific
+terms of ownership should be define by those members as part of their application
+for the adoption of the idea under BIFFUD.
+
+C. **BIFFUD Core:** projects that are taken on by the entire corporation.  These
+might be projects utilized by the Corporate Overlords to execute on company policy,
+or simply be projects that the Corporate Overlords decide to develop without a concept
+of individual ownership.
+
+
+### Evaluating Ideas
+Anybody can build anything they want at any time, but in order for something to become
+an official BIFFUD project it must be formally adopted by the Corporate Overlords during
+a plotting session.  The determination of what makes a project a BIFFUD project is
+ultimately determined by that process, but in the spirit of framing the conversation,
+the following questions are recommended.
+
+1. How is this project a bad idea?
+2. Does it make you thinking face emoji?
+3. Is it realistic to implement through BIFFUD?
+4. If the project were a character in Dungeons and Dragons, what would it's alignment be?
+
+The BIFFUD brand deserves compelling answers to questions 1 and 2. Projects should be
+realistic to implement, and should either be "Chaotic Good", "Neutral Good", or
+"Chaotic Neutral".  All of these data points simply serve as guidance. The ultimate
+decision of what projects should be adopted falls on the Corporate Overlords.
+
+
+### Intellectial Property / Internet Protocol
+The following numbered clauses apply to all BIFFUD projects unless explicitly approved
+otherwise by the Corporate Overlords during project approval or at a later date during
+a plotting session.
+
+1. Everything contributed to a BIFFUD project must be developed in the open and licensed under
+Apache 2.0.
+
+2. All BIFFUD projects must have the language written in `documents/contributing_addition.md`
+inserted into a CONTRIBUTING.md file at the root level of the project unless explicitly
+approved otherwise by the Corporate Overlords during project approval or at a later date
+during a plotting session.
+
+3. Copyright for all potential code or other creative works with a BIFFUD project must be
+granted to BIFFUD before it can be incorporated into a project, unless the Beastmaster gives
+their approval.
```

Comment:
again, i would like to add patent stuff:

> Copyright and patent rights for all potential code or other creative works within a BIFFUD project
> must be granted to BIFFUD before it can be incorporated into a project, unless the Beastmaster
> gives their approval.

(I think the difference between this and relying on Apache License is who owns the "invention". This gives it all to BIFFUD so BIFFUD can patent the stuff; latter leaves ownership in the hands of the contributor but protects BIFFUD/contributors/users of the project & their ability to use/sell/distribute/etc the project forever even if contributor patents what they added to the project.)

In any case, the Beastmaster can override all this stuff.